src/sentry/integrations/slack/utils.py

```diff
@@ -18,8 +18,15 @@
 logger = logging.getLogger('sentry.integrations.slack')
 
 # Attachment colors used for issues with no actions take
-NEW_ISSUE_COLOR = '#E03E2F'
```

Comment:
Could you get rid of `NEW_ISSUE_COLOR` and then down where we set the color just default to `LEVE_TO_COLOR['error']` if there is no level.